docs/BreakingChanges/151 DataGridCellsPanel.BringIntoView throws ArgumentOutOfRangeException.md

```diff
@@ -0,0 +1,37 @@
+## 151: DataGridCellsPanel.BringIndexIntoView throws ArgumentOutOfRangeException
```

Comment:
There already exists a different breaking change with the ID 151. https://raw.githubusercontent.com/Microsoft/dotnet-apiport/master/docs/BreakingChanges/151%20Item-scrolling%20a%20flat%20list%20with%20items%20of%20different%20pixel%20height.md